src/SW_netCDF.c

```diff
@@ -0,0 +1,591 @@
+    write_att_str("units", "1", *domID, LogInfo);
+}
+
+/**
+ * @brief Create the "crs" variable in a netCDF
+ *
+ * @param[in] LogInfo Holds information dealing with logfile output
+*/
+void create_var_crs(LOG_INFO* LogInfo) {
+    int crsID;
+    double stdParallel[] = {29.5, 45.5}, longCentralMerid[] = {-96.0},
+        latProjOrigin[] = {23.0}, false_easting[] = {0.0},
+        false_northing[] = {0.0}, longPrimeMerid[] = {0.0},
+        semiMajorAxis[] = {6378137.0}, inverse_flattening[] = {298.257222101};
+
+    // Constant attribute strings
+    static char *crs_wktVal = "PROJCS[\"NAD83(2011) / Conus Albers\","\
+        "GEOGCS[\"NAD83(2011)\",DATUM[\"NAD83_National_Spatial_Reference"\
+        "_System_2011\",SPHEROID[\"GRS 1980\",6378137,298.257222101]],PRIMEM"\
+        "[\"Greenwich\",0],UNIT[\"degree\",0.0174532925199433,AUTHORITY"\
+        "[\"EPSG\",\"9122\"]],AUTHORITY[\"EPSG\",\"6318\"]],PROJECTION"\
+        "[\"Albers_Conic_Equal_Area\"],PARAMETER[\"latitude_of_center\",23],"\
+        "PARAMETER[\"longitude_of_center\",-96],PARAMETER[\"standard_parallel"\
+        "_1\",29.5],PARAMETER[\"standard_parallel_2\",45.5],PARAMETER[\""\
+        "false_easting\",0],PARAMETER[\"false_northing\",0],UNIT[\"metre\",1],"\
+        "AXIS[\"Easting\",EAST],AXIS[\"Northing\",NORTH],AUTHORITY[\"EPSG\","\
+        "\"6350\"]]";
+
+    // Create "crs" and write attributes
+    create_variable("crs", ZERODIMS, NULL, NC_INT, &crsID, LogInfo);
+
+    write_att_str("grid_mapping_name", "albers_conical_equal_area",
+                  crsID, LogInfo);
+
+    write_att_val("standard_parallel", stdParallel, SIZETWO, crsID, LogInfo);
+    write_att_val("longitude_of_central_meridian", longCentralMerid, SIZEONE,
+                  crsID, LogInfo);
+    write_att_val("latitude_of_projection_origin", latProjOrigin, SIZEONE,
+                  crsID, LogInfo);
+    write_att_val("false_easting", false_easting, SIZEONE, crsID, LogInfo);
+    write_att_val("false_northing", false_northing, SIZEONE, crsID, LogInfo);
+    write_att_val("longitude_of_prime_meridian", longPrimeMerid, SIZEONE,
+                  crsID, LogInfo);
+    write_att_val("semi_major_axis", semiMajorAxis, SIZEONE, crsID, LogInfo);
+    write_att_val("inverse_flattening", inverse_flattening, SIZEONE,
+                  crsID, LogInfo);
+
+    write_att_str("crs_wkt", crs_wktVal, crsID, LogInfo);
+}
+
+/**
+ * @brief Fill variables within domain "xy" with values
+ *
+ * @param[in] nDimX Size of dimension "x"
+ * @param[in] nDimY Size of dimension "y"
+ * @param[in] domainID ID of the domain variable within the domain netCDF
+ * @param[in] xID ID of the x variable within the domain netCDF
+ * @param[in] yID ID of the y variable within the domain netCDF
+ * @param[in] x_bndsID ID of the x_bnds variable within the domain netCDF
+ * @param[in] y_bndsID ID of the y_bnds variable within the domain netCDF
+*/
+void fill_xy_vars(int nDimX, int nDimY, int domainID,
```

Comment:
`suids`, i.e., domain, and x and y or s coordinates are calculated by "SW_domain" if needed. `fill_xy_vars()` or `fill_s_vars()` should just write coordinate values into their matching variables.